tests/plugin/text/slot_filler/test_rule_slot_filler.py

```diff
@@ -371,6 +373,7 @@ def test_slot_filling_order() -> None:
         score=0.5,
         entity_type="entity_1",
         values=[{"value": "value_3"}],
+        alternative_index=0
```

Comment:
This test is only testing if sorting by score works well. Sorting by alternative_index (which was the reason why we needed this PR) is not being tested. Please add that.